map_data/state_regions/15_russia.txt

```diff
@@ -0,0 +1,1276 @@
+    impassable = {}
+    traits = { "state_trait_scandinavian_forests" }
+    city = "x91984B"
+    port = "xF2B2C8"
+    farm = "x57190F"
+    mine = "xE229D5"
+    wood = "xA09A15"
+    arable_land = 2
+    arable_resources = { "bg_rye_farms" "bg_livestock_ranches" }
+    capped_resources = {
+        bg_iron_mining = 85
+        bg_sulfur_mining = 10
+        bg_logging = 25
+        bg_fishing = 10
+    }
+    naval_exit_id = 3039
+}
+
+STATE_INGRIA = {
+    id = 131
+    subsistence_building = "building_subsistence_farms"
+    provinces = { "x01A0E0" "x042A4F" "x065E68" "x0FEA4B" "x246095" "x3F6DEC" "x441576" "x57DF51" "x607A9E" "x6122C5" "x743B9C" "x80A0E0" "x8F79E9" "x93FBC9" "xA79D26" "xAD3522" "xAEBDCB" "xB64167" "xB6C9C3" "xBB73A3" "xCD164E" "xD14E99" "xD1CBDD" "xD6A28B" "xD7689F" "xDA2E04" "xDED975" "xE97616" "xE9C671" "xF4E44C" }
+    traits = { "state_trait_russian_forest" "state_trait_cold_water_port" "state_trait_st_petersburg" "state_trait_kronstadt_island" }
+    city = "xCD164E"
+    port = "xCD164E"
+    farm = "xDED975"
+    mine = "xB64167"
+    wood = "x80A0E0"
+    arable_land = 50
+    arable_resources = { "bg_rye_farms" "bg_livestock_ranches" }
+    capped_resources = {
+        bg_logging = 25
+        bg_fishing = 10
+    }
+    naval_exit_id = 3000
+}
+
+STATE_MOSCOW = {
+    id = 132
+    subsistence_building = "building_subsistence_farms"
+    provinces = { "x13D91C" "x43F336" "x4DBAEC" "x54C962" "x632D29" "x8B324E" "xC05614" "xE0A021" "xE19E7E" "xE32FD0" "xEC68C6" "xF75BCA" }
+    traits = { "state_trait_volga_river" "state_trait_russian_forest" "state_trait_moscow" }
+    city = "x13D91C"
+    farm = "x54C962"
+    mine = "x8B324E"
+    wood = "xE19E7E"
+    arable_land = 60
+    arable_resources = { "bg_rye_farms" "bg_livestock_ranches" }
+    capped_resources = {
+        bg_coal_mining = 60
+        bg_iron_mining = 30
+        bg_logging = 34
+        bg_monuments = 1
+    }
+}
+
+STATE_BREST = {
+    id = 133
+    subsistence_building = "building_subsistence_farms"
+    provinces = { "x157352" "x18020B" "x1917E9" "x241A22" "x406020" "x66FB56" "x6D7F7E" "x75F1B4" "x7E7F58" "x806161" "x84294C" "x84FE3D" "x862F77" "x8ABE62" "x936BF9" "xA201E3" "xA6BED3" "xA93262" "xAB3AC5" "xAC6FAE" "xB338E8" "xBCB94C" "xBFDF20" "xC15AD8" "xC1D9C0" "xC3FA73" "xCB21FD" "xCCD197" "xDE471E" "xE06176" "xE87D11" "xF02643" }
+    traits = { "state_trait_russian_forest" }
+    city = "x8ABE62"
+    farm = "xAB3AC5"
+    mine = "x157352"
+    wood = "xA6BED3"
+    arable_land = 75
+    arable_resources = { "bg_rye_farms" "bg_livestock_ranches" }
+    capped_resources = {
+        bg_logging = 14
+    }
+}
+
+STATE_VOLHYNIA = {
+    id = 134
+    subsistence_building = "building_subsistence_farms"
+    provinces = { "x0EB3C7" "x1A9FFE" "x364496" "x37A636" "x5A26AE" "x71E0A0" "x80FB0F" "x8201CF" "xAC3079" "xAFEBC8" "xE001E0" "xE2F553" }
+    traits = { "state_trait_russian_forest" "state_trait_black_soil" }
```

Comment:
any reason to differentiate between state_trait_black_soil and state_trait_chestnut_soils?

Reply:
Same as with the North Andes